.github/workflows/vm-build.yml

```diff
@@ -105,3 +105,47 @@ jobs:
         working-directory: ${{github.workspace}}
 
       - run: npm run test:blockchain
+
+  vm-benchmarks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: actions/checkout@v2
+
+      - name: Dependency cache
+        uses: actions/cache@v2
+        id: cache
+        with:
+          key: VM-${{ runner.os }}-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+          path: '**/node_modules'
+
+      # Installs root dependencies, ignoring Bootstrap All script.
+      # Bootstraps the current package only
+      - run: npm install --ignore-scripts && npm run bootstrap:vm
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Just out of interest: what is this line doing respectively how does this behave?

Reply:
installs deps and builds the vm for use in the benchmarks in the next steps

Reply:
Ah no, was just referring to the `if`-clause: `if: steps.cache.outputs.cache-hit != 'true'` 😄

Reply:
oh :) that’ll skip the install step if it was able to restore the deps from the cache step above it